kyverno/manifest.yaml

```diff
@@ -6,3 +6,20 @@ maintainer: "Kyverno Authors"
 description: Kyverno is a policy engine designed for Kubernetes
 url: https://kyverno.io
 category: security
+plans:
+  - label: "Default Policy Security Standard in audit mode"
+    configuration:
+      POD_SECURITY_STANDARD:
```

Comment:
would be good to rename to `POD_SECURITY_STANDARDS`

Reply:
changed